File: downtimeC/InputBoxForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using Microsoft.VisualBasic;
using System.Collections;
using System.Diagnostics;

namespace downtimeC
{
    public partial class InputBoxForm : Form
    {
        public InputBoxForm()
        {
            InitializeComponent();
        }

    private void InputBoxForm_Load(object sender, System.EventArgs e)
    {
        this.TopMost = true;
        this.Activate();

        string a = "Please enter correct test for:" + " " + RecoveryForm.tests;


        this.LabelTestError.Text = a;


        this.TextBoxTestFix.Focus();
        //InputBoxFormRecover.TextBoxTestFix.Text = b
        //Me.label()


    }


    private void ButtonInput_Click(object sender, System.EventArgs e)
    {
        RecoveryForm.fixTEST = TextBoxTestFix.Text;
        this.Close();
    }
    public void label()
    {
        string a = "Please enter correct test for:";
        string b = a + " " + RecoveryForm.tests;
        this.LabelTestError.Text = b;
        Console.WriteLine(this.LabelTestError.Text);
    }



    private void TextBoxTestFix_KeyPress(object sender, System.Windows.Forms.KeyPressEventArgs e)
    {
        if (e.KeyChar == Strings.Chr(13)) {
            ButtonInput_Click(this, EventArgs.Empty);
        }

    }

    }
}

File: downtime/InputBoxForm.vb
Public Class InputBoxFormRecover

    Private Sub InputBoxForm_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles Me.Load

        Me.TopMost = True
        Me.Activate()

        Dim a As String = "Please enter correct test for:" + " " + RecoveryForm.tests


        Me.LabelTestError.Text = a


        Me.TextBoxTestFix.Focus()
        'InputBoxFormRecover.TextBoxTestFix.Text = b
        'Me.label()


    End Sub


    Private Sub ButtonInput_Click(ByVal sender As Object, ByVal e As System.EventArgs) Handles ButtonInput.Click
        RecoveryForm.fixTEST = TextBoxTestFix.Text
        Me.Close()
    End Sub
    Sub label()
        Dim a As String = "Please enter correct test for:"
        Dim b As String = a + " " + RecoveryForm.tests
        Me.LabelTestError.Text = b
        Console.WriteLine(Me.LabelTestError.Text)
    End Sub



    Private Sub TextBoxTestFix_KeyPress(ByVal sender As Object, ByVal e As System.Windows.Forms.KeyPressEventArgs) Handles TextBoxTestFix.KeyPress
        If e.KeyChar = Chr(13) Then
            ButtonInput_Click(Me, EventArgs.Empty)
        End If

    End Sub

    
End Class